source/constructs/api/catalog/service.py

```diff
@@ -295,7 +297,7 @@ def sync_crawler_result(
                 database_column_count += column_order_num
             try:
                 logger.info("batch delete glue tables" + json.dumps(delete_glue_table_names))
-                client.batch_delete_table(DatabaseName=glue_database_name,
+                glue_client.batch_delete_table(DatabaseName=glue_database_name,
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code makes a call to a batch operation, but it does not handle any errors returned by that batch operation. Batch operations can succeed without throwing exceptions even if some items in the request fail. Consider checking the return value of the batch operation for failures.

[Learn More](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue.html#Glue.Client.batch_delete_table)